mantidproject/mslice
# Set intensity from action type

**Description of work:**
This PR fixes a crash caused by caching a string rather than an action object. This causes a crash when calling `setChecked` on the string rather than on the action object.

**To test:**
1. Create a cut plot
2. change intensity to Chi"(Q,E) but click on cancel
3. There should be no crash. The intensity should remain the same

Part of #854
